src/components/atoms/DateTimePicker/Component.tsx

```diff
@@ -0,0 +1,275 @@
+import React, { FunctionComponent, forwardRef, useState } from 'react';
+import {
+  TouchableOpacity,
+  StyleSheet,
+  View,
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  Modal,
+} from 'react-native';
+import RNDateTimePicker from '@react-native-community/datetimepicker';
+import type { PropsWithRef } from '../../../type';
+import { defaultTheme } from '../../../theme';
+import { Text, Icon } from '../../../index';
+
+/**
+ * DateTimePicker support date/time select on iOS and Android and built on top of react-native-datetimepicker
+ * Reference: https://github.com/react-native-datetimepicker/datetimepicker
+**/
+
+type DateMode = 'date' | 'time' | 'datetime'
+
+export type DateTimePickerProps = PropsWithRef<{
+  value?: Date;
+  dateMode?: DateMode;
+  confirmText?: string;
+  cancelText?: string;
+  onChange?: (date: any) => void;
+  onBlur?: any;
+  style?: ViewStyle;
+  placeholder?: string;
+  showIcon?: boolean;
+  disabled?: boolean;
+  iconComponent?: JSX.Element;
+  placeholderStyle?: TextStyle;
+  cancelButtonStyle?: TextStyle;
+  confirmButtonStyle?: TextStyle;
+  locale?: string;
+  timeZoneOffsetInMinutes?: number;
+  textColor?: string;
+  themeVariant?: 'dark' | 'light';
+  maximumDate?: Date;
+  minimumDate?: Date;
+  display?: 'default' | 'compact' | 'inline' | 'spinner' | 'clock' | 'calendar';
+  errorStyle?: StyleProp<TextStyle>;
+  errorMessage?: string;
+  renderErrorMessage?: boolean;
```

Comment:
Please remove all of these props :D , in my issues description already provide props we need, so let delete all unused props

Reply:
Yes, I checked and remove some. I think the rest will be needed for some cases like using Form, custom mode/display and style. Are you sure we will remove all of that?

Reply:
Ya, in issue we just need some props is a must, 

<img width="532" alt="Screen Shot 2022-01-26 at 11 52 02 AM" src="https://user-images.githubusercontent.com/7247063/151105899-389f277a-1197-4576-acd8-e2d20149c399.png">